uwuno/src/main/java/com/learning/uwuno/room.java

```diff
@@ -33,12 +45,41 @@ public void setRoomName(String roomName) {
         this.roomName = roomName;
     }
 
+    public void setRoomStatus(Status status) {
+        this.roomStatus = status;
+    }
+
+    public Status getRoomStatus() {
+        return roomStatus;
+    }
+
     public String getUid() {
         return uid;
     }
 
+    public deck getDeck() {
+        return deck;
+    }
+
+    @JsonIgnoreProperties("cardList")
```

Comment:
Is this on purpose? Are we trying to ignore cardList or playerTurn?

Reply:
Yes, we're ignoring the cardList field for playerTurn in the response JSON so it doesn't clog it (and repeat the same card list information shown in playerList). We only need the player id and name to tell who the player is.

Reply:
Oops I meant to say what you just said should be in a comment.